HelpSource/Guides/WritingClasses.schelp

```diff
@@ -32,72 +37,147 @@ MyClass : SomeSuperclass {
 Without specifying a superclass, link::Classes/Object:: is assumed as the default superclass.
 
 code::
-MyClass { // :Object is implied
+MyClass { // : Object is implied
 
 }
 ::
 
+This is why in the above example, the message code::new:: can be called without being explicitly defined in the class.
+
+
+
 section:: Methods
 
-class methods are specified with the asterisk within the class method, the keyword code::this:: refers to the class. A
-class in SuperCollider is itself an object.  It is an instance of link::Classes/Class::. Instance methods are specified
-within the instance method, the keyword code::this:: refers to the instance.
+subsection:: Instance Methods
+
+Each object instance responds to its strong::instance methods::.Within instance methods, the keyword code::this:: refers to the instance itself.
+
+An instance is independent of all the other instances, unless otherwise specified.
```

Comment:
I'm not sure what this means exactly.. what does it mean to be independent w/r/t instance methods, and when specifically does that change? Perhaps this would be better explained in another section, I get the feeling that it relates more to instance variables than to instance methods

Reply:
I hope it's better now.